src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -376,35 +236,14 @@ public List<Map<String, Object>> fetchTableRecordsRetryableMode(String tableName
    */
   @Nullable
   public Schema fetchServiceNowTableSchema(String tableName, FailureCollector collector) {
```

Comment:
seems this can be replaced by `fetchTableSchema`.

Reply:
Renamed the method. 093d31e47a15f475e4d355bcbbfcb76d1c71e158